crypt_keeper.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = CryptKeeper::VERSION
 
-  gem.add_runtime_dependency 'activerecord',  '>= 3.1', '< 4.2'
-  gem.add_runtime_dependency 'activesupport', '>= 3.1', '< 4.2'
+  gem.add_runtime_dependency 'activerecord',  '>= 3.1', '<= 4.2'
+  gem.add_runtime_dependency 'activesupport', '>= 3.1', '<= 4.2'
```

Comment:
@Ch4s3 is there a specific reason you went with `<=` over `< 4.3`? If not, I'm going to change that in master after merging.